### DISCUSSION TITLE
Vscode

### DISCUSSION THREAD

**Discussion by anthonyjoelym2000** [community/community#125855] [Programming Help]
### Select Topic Area

Question

### Body

Una consulta  soy nuevo en esto de Vscode pero tengo  un problema se que con el tiempo mejorare  mi ingles pero quisiera saber si existe una extensión que pueda visualizar el significado de las palabras, de las opciones y que me salga en ESPAÑOL ósea quiero todo el VSC este en ingles por defecto pero si pongo el mouse encima de alguna opción o código me vote como una ventana emergente el significado en español existirá alguna extensión  ? para mas o menos sabe que significa en español

**Comment by davevad93** [comment]
Hi @4nj04444 ,

I'm not aware of the existence of an extension like that in VS Code, the only extensions I know for Spanish are the [Spanish Language Pack for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-es) which provides a localized UI experience in `Español` and [Spanish - Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-spanish) which imports the Spanish spell checking dictionary for [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @anthonyj03l ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hi @anthonyj03l, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
